frontend/main.js

```diff
@@ -280,8 +283,21 @@ async function startApp () {
       this.setBadgeOnTab()
     },
     computed: {
-      ...mapGetters(['ourUnreadMessages', 'totalUnreadNotificationCount']),
+      ...mapGetters(['groupsByName', 'groupProposals', 'ourUnreadMessages', 'totalUnreadNotificationCount']),
       ...mapState(['contracts']),
+      groupOpenProposalsByGroup () {
+        return this.groupsByName.map(group => {
+          const { contractID } = group
+          const proposals = {}
+          for (const proposalId in (this.groupProposals(contractID) || {})) {
+            const proposal = this.groupProposals(contractID)[proposalId]
```

Comment:
Instead of calling `this.groupProposals` twice, you can cache the value outside of the loop.

Reply:
Yeah. Make sense.